scripts/linux/create_concord_clients_transaction_signing_keys.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# This script generates sets of key pairs - private/public keys to be used for
+# concord client (participant node) transaction signing.
+#
+# Output is generated into output path output_base_path/transaction_signing_keys folder.
+# If --output_base_path is not given by user, output is generated into output
+# path script_working_path/transaction_signing_keys.
+# Each generated set resides in a separate numeric sub-folder, range from 1 to
+# num_participants.
+#
+# In general, it can be used by any 2 entities; the signer should use the
+# private_key.pem to sign, and the verifier should use public_key.pem
+# to verify the signature.
+
+# This script assumes that the output path is empty. If not, content is deleted.
```

Comment:
That seems dangerous

Reply:
OK, I'm going to add an option to overwrite an existing folder.
Else, if the destination folder is not-empty, the script will fail.
Thanks!

Reply:
@andrewjstone  thanks for the comment, I'll just fail the script if the base destination folder is not empty, and print a warning.

Reply:
Cool! Approved :)